packages/formatting/presets/relaxed.yaml

```
@@ -0,0 +1,63 @@
+# @outfitter/formatting Relaxed Preset
+# Flexible formatting for rapid development
+name: relaxed
+description: Relaxed formatting rules for rapid prototyping and flexible development
+
+common:
+  # Longer lines allowed
+  lineWidth: 120
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)